wagtail/wagtailcore/tests/test_blocks.py

```diff
@@ -1447,6 +1447,34 @@ class ShoppingListBlock(blocks.StructBlock):
         )
         self.assertIn('value="chocolate"', form_html)
 
+    def test_max_length_validation_error(self):
```

Comment:
Could you also add some tests for ListBlocks with a max_length/min_length where validation passes?